lms/djangoapps/grades/models.py

```diff
@@ -23,6 +23,8 @@
 log = logging.getLogger(__name__)
 
 
+BLOCK_RECORD_VERSION = 1
```

Comment:
Nit: should this be `BLOCK_RECORD_LIST_VERSION`?  Since it versions the list and not each individual record in it.